src/components/AddNewBook.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books';
+import './addNewBook.css';
+
+const AddNewBook = () => {
+  const dispatch = useDispatch();
+  const options = ['Fantasy', 'Philosophy', 'ScienceFiction'];
+  const category = options.map((o) => (
+    <option key={o}>{o}</option>
+  ));
+
+  return (
+    <div className="new-book">
+      <div>
+        <h3>Add new Book</h3>
+      </div>
+      <div>
+        <form
+          onSubmit={(e) => {
+            const form = document.querySelector('FORM');
+            e.preventDefault();
+            const book = {
+              title: form.title.value,
+              author: form.author.value,
+              category: form.category.value,
+              pages: 100,
+              completed: 0,
+              chapter: 0,
+            };
+
+            dispatch(addBook(book));
+          }}
```

Comment:
**Changes Required**
- [ ] Good effort handling onSubmit with an anonymous function. I would advice that you refactor this into a `named` ES6 arrow function and just call it here. This will make your code much more readable and testable.